core/src/install.js

```diff
@@ -134,4 +136,6 @@ window.addEventListener('DOMContentLoaded', function() {
 		drawTitles: true,
 		nonce: btoa(getToken()),
 	})
+
+	$("#dbpass").showPassword().keyup()
```

Comment:
Nice catch, let's also re-add the missing toggle for the admin password then from https://github.com/nextcloud/server/commit/bba26cb2ebf48413ed938af572ee46b7cb9f0ba2#diff-e60184f15246b33f90d369d78f0f2df1L3

```suggestion
	$('#adminpass').showPassword().keyup()
	$("#dbpass").showPassword().keyup()
```

Reply:
The show button is not rendered correctly on the adminpass field. I will add a commit that fixes the template too. Just adding this line won't do anything.